tests/components/upnp/test_config_flow.py

```diff
@@ -442,3 +444,40 @@ async def test_flow_user_no_discovery(hass: HomeAssistant) -> None:
     )
     assert result["type"] == data_entry_flow.FlowResultType.ABORT
     assert result["reason"] == "no_devices_found"
+
+
+@pytest.mark.usefixtures(
+    "ssdp_instant_discovery",
+    "mock_setup_entry",
+    "mock_get_source_ip",
+    "mock_mac_address_from_host",
+)
+async def test_flow_ssdp_with_mismatched_udn(hass: HomeAssistant) -> None:
+    """Test config flow: discovered + configured through ssdp, where the UDN differs in the SSDP-discovery vs device description."""
```

Comment:
Please limit docstring lines to 72 characters per line, in accordance with PEP8.

The first line is a header and should be a single sentence. After that an optional body can follow separated from the header by a blank line.